src/lib/version/version.h

```diff
@@ -99,12 +99,41 @@ static inline const char *px4_build_uri(void)
 }
 
 /**
+ * Convert a version tag string to a number
+ * @param tag version tag in one of the following forms:
```

Comment:
I assume the vendor form `vendor: v1.4.0-0.2.0` is valid here too?

Reply:
Sure, I've added an another comment for this.